crates/topos-tce-api/src/runtime/mod.rs

```diff
@@ -222,6 +225,36 @@ impl Runtime {
 
     async fn handle_internal_command(&mut self, command: InternalRuntimeCommand) {
         match command {
+            InternalRuntimeCommand::NewTransientStream { sender } => {
+                let stream_id = Uuid::new_v4();
+                info!("Opening a new transient stream with UUID {stream_id}");
+
+                let (stream, receiver) = mpsc::channel(TRANSIENT_STREAM_CHANNEL_SIZE);
+                let (shutdown, shutdown_recv) = oneshot::channel();
+                self.transient_stream.insert(stream_id, stream);
+
+                self.streams.push(
+                    shutdown_recv
+                        .map_err(move |_| StreamError {
+                            stream_id,
+                            kind: StreamErrorKind::StreamClosed,
+                        })
+                        .boxed(),
+                );
+
+                if sender
+                    .send(Ok(TransientStream {
+                        stream_id,
+                        inner: receiver,
+                        notifier: Some(shutdown),
+                    }))
+                    .is_err()
+                {
+                    error!("Unable to send new TransientStream");
```

Comment:
Print the error too perhaps?

Reply:
The error on a `oneshot::Sender` is `T` so a `TransientStream`. It has no value to print it.